docs/source/release_notes.rst

```diff
@@ -228,7 +229,7 @@ v1.0.0 Oct 12, 2021
         * Update API Reference to match Featuretools 1.0 API (:pr:`1600`)
         * Update Index page to be Jupyter notebook and use Woodwork integration (:pr:`1602`)
         * Update Feature Descriptions page to be Jupyter notebook and use Woodwork integration (:pr:`1603`)
-        * Update Using Koalas EntitySets page to be Jupyter notebook and use Woodwork integration (:pr:`1604`)
```

Comment:
I don't think we need to retroactively replace Koalas with Spark in the changelog

Reply:
yep agreed, I missed that change possibly after merging conflicts locally